remill/Arch/AArch64/Semantics/DATAXFER.cpp

```diff
@@ -208,6 +208,20 @@ DEF_SEM(MoveWithKeep, D dst, S src, I64 imm, I8 shift_) {
   return memory;
 }
 
+// imm has already been converted to float representation
+// but is an uint_n type, therefore we don't attempt
+// any float conversion right
+DEF_SEM(FloatMovImm32, V128W dst, I32 imm) {
```

Comment:
I think I kind of would like to see this be a `F32`  (and `F64` below) if you can make it work how we discussed, and then using `FWriteV32` (and `FWriteV64` below). It's a bit of extra work, but it'll make the resulting bitcode look a bit cleaner because it won't have the bitcast going on.